dapps/marketplace/src/pages/nav/GetStarted.js

```diff
@@ -1,23 +1,25 @@
 import React from 'react'
 
 import Avatar from 'components/Avatar'
+import UserActivationLink from 'components/UserActivationLink'
+import { withRouter } from 'react-router-dom'
 
-const GetStarted = ({ onClick }) => (
+const GetStarted = withRouter(({ onClick, location }) => (
   <ul className="navbar-nav">
     <li className="nav-item">
-      <a
+      <UserActivationLink
         className="nav-link"
-        href="#get-started"
-        onClick={e => {
-          e.preventDefault()
+        to={`/onboard`}
+        onClick={() => {
           if (onClick) onClick()
         }}
+        location={location}
```

Comment:
can we move withRouter to the UserActivationLink component?

Reply:
In certain places we may want to override the `location` prop. Like in the components `_CreateListingCard.js`, `_EarnTokensCard.js` and `pages/nav/Profile.js`. Also the "Contact Seller/User" links in listing detail page and user profile page.

That's the reason I didn't add that HOC to `UserActivationLink` component.

Reply:
Merging this PR now. But if this change has to be done, let me know. I'll have another PR addressing it.